ONSdigital/sixteens
# Fix - Overflow shadows tables and equations

### What
Second part of adding scrollable overflow. This PR adds support in:
1. Equations
1. V1 and V2 tables
1. Removes any uses of the "view table" button
1. Remove commented out code that kept coming up in searches!

### How to review
1. Visit a bulletin or article with any of the above
1. See that they overflow the viewport on small screens and cause the page to scroll
1. Switch to this branch and babbage branch `fix/overflow-shadows-2`
1. See that they now scroll within the viewports with shadows indicating when there is content horizontally available

### Who can review
Anyone but me
